### ISSUE TITLE
Some file extensions/mime types are not preserved for downloads

### ISSUE THREAD

**Issue by DaniFoldi** [outline/outline#7000]
**To Reproduce**
1. Upload a file, in my example I used an illustrator file, with .ai extension.
2. Download the file
3. Observe that the downloaded file doesn't have the original file's name, and has .ps extension

**Expected behavior**
I believe .ai files should retain their extensions. I've also noticed that the file names of images are also changed to eg. attachment.png when downloading.

**Screenshots**

**Outline (please complete the following information):**
- Install: self hosted
- Version: 4cb0f8b (0.76.1) via Docker Hub image

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: tested in Chrome 125 and Safari 17.5

**Comment by tommoor** [comment]
I cannot reproduce yet, what file storage are you using?

**Comment by DaniFoldi** [comment]
Hey Tom,

Thank you for the quick response! We use MinIO, with an extra-convoluted proxy setup to avoid CORS, since our instance is behind Cloudflare Access, and wouldn't work with API requests going to a different application.

Upon further investigation, I found that MinIO was only setting `content-disposition: attachment`, which browsers then interpreted freely, leading to attachment.redirect.ps, attachment.png and so on. I added an extra line to our proxy that adds the last part of the filename to the content-disposition header: ```js
const disposition = originalDisposition.includes('filename') ? originalDisposition : `${originalDisposition}; filename=${redirectUrl.pathname.split('/').pop()}`;
```

which seems to have resolved the issue.
